set up for dev (#11)
votingworks/vxsuite-complete-system: README.md

compatibility and versioning. This repository has all the components
and scripts to run each machine (BMD, BAS, BSD, EMS).

## Hardware and OS

> Pick a machine with FHD resolution and the ability to install Linux.

Reviewer: What's FHD?

Author: ```suggestion
Pick a machine with FHD (1920x1080) resolution and the ability to install Linux.
```

Install Ubuntu 18.04.4+ Desktop, minimal installation. Software update.

## Install VotingWorks Software

```
sudo apt install git make
git clone https://github.com/votingworks/vxsuite-complete-system
cd vxsuite-complete-system
make node
```

> Then pull down all of the code. You can rerun these commands to pull

Reviewer: ```suggestion
Then pull down all of the code. You can re-run these commands to pull
```

the latest version of the code.

```
make checkout
make build
```

## Run in Development

You can now run the services needed for any of the systems as follows:

```
bash run-election-manager.sh
```

```
bash run-bsd.sh
```

```
bash run-bmd.sh
```

```
bash run-bas.sh
```

Once the services are running, start the Kiosk Browser:

```
bash run-kiosk-browser.sh
```

You're good to go. You can exit the Kiosk Browser with Ctrl-W.

## Configuring for Production

> To configure and lock down the machine for production use, nothing

Reviewer: ```suggestion
To configure and lock down the machine for production use, noting
```
